### PR TITLE
[ADP-3303] Fix quickstart instruction in the README

### PR DESCRIPTION
**Pull request by paolino**
- [x] Fix snapshot URLs

Solves #4458

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,59 +56,69 @@ start with Daedalus.
...
 
-# start the services
-docker-compose up -d
 
-# work with the wallet with icarus frontend
-#    (not a production tool, DO NOT USE IT WITH mainnet)
-# connect to http://localhost:4444
+if [ "$(ls -A "${NODE_DB}")" ]
+then
+    echo "Node state is present, not downloading the snapshot."
```
Comment (paweljakubas) [inline README.md:95]:
```suggestion
    echo "The requested node version is present, not downloading the snapshot."
```

File: configs/cardano/preprod/genesis.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<html>
```
Comment (paweljakubas) [inline configs/cardano/preprod/genesis.json]: are we sure this file is needed?

**Review by paweljakubas — APPROVED** [review]
nice! works for me!

File: configs/cardano/preprod/genesis.json (hunk 2 above)
Comment (paolino) [reply configs/cardano/preprod/genesis.json]: removed

**Pull request merged**